src/translator/translator.js

```diff
@@ -64,27 +65,64 @@ class GoogleTranslator {
 
       return response.data.data.translations[0];
     } catch (e) {
-      if (e.response) {
-        const err = new GoogleApiError(e.response);
-        logger.error(
-          `Google Translation API Error: ${JSON.stringify(
-            err.details,
-            null,
-            2
-          )}`
-        );
-        logger.error(err);
-        throw err;
-      } else {
-        logger.error(`Google Translation API Error: ${e.message}`);
-        throw e;
-      }
+      this._handleAxiosError(e);
     }
   }
 
   _getISOLanguageCode(language) {
     return ISO6391.getCode(language) || language; // if `language` was already an ISO-639-1 code, getCode() will return '' and return value will be defaulted to `language` itself
   }
+
+  _getISOLanguageName(code) {
+    return {
+      name: ISO6391.getName(code) || '',
+      native: ISO6391.getNativeName(code) || '',
+    }; // if `language` was already an ISO-639-1 code, getCode() will return '' and return value will be defaulted to `language` itself
+  }
+
+  async _getSupportedLanguages() {
```

Comment:
You can remove the `_` prefix from this. It's a public function meant to be called from outside the class.

I think my code was a little confusing so let me explain the pattern I was following. If a class function is to be public, i don't prefix it with an `_`. If it's supposed to be private, i prefix it with an `_`.

Only doing this because JS doesn't really have private class methods. If this was in TypeScript I'd just use `private` and `public`